docs/getting-started.md

```diff
@@ -1,8 +1,21 @@
-## [WIP] Getting started with wrangler
+## Getting started with 🤠 Wrangler
```

Comment:
```suggestion
## Getting started with ⛅️ Wrangler
```
I hate that stipid emoji lol

Reply:
hahahah I ONLY copied it from the main readme.md 🤣